### DISCUSSION TITLE
Viewing multiple images in TensorBoard 3D

### DISCUSSION THREAD

**Discussion by swahtz** [Project-MONAI/MONAI#4722] [Q&A]
Hello,

I've just started using the TensorBoard 3D plugin and really liking it, cheers!  I am having one issue figuring out how to cycle through (or display side-by-side) the multiple images that I've logged during training.  I've used MONAI's plot_2d_or_3d_image to log three 3D images with different tags during training which I can see as animated images in TensorBoard:
![image](https://user-images.githubusercontent.com/2375296/179734369-219e7cdc-d289-4bf8-a67a-3d60d08ec510.png)

But when using the TensorBoard 3D plugin, I can only see one of the images and I can't figure out how to switch the plugin to render one of the other images (or series of images I'm logging) with a different tag than the one that shows by default:

![image](https://user-images.githubusercontent.com/2375296/179735189-8caa419e-b236-4621-ada6-d8d410c9a401.png)

Is there any advice on how to switch between images?

Thank you

**Comment by KumoLiu** [comment]
Hi @swahtz,
`plot_2d_or_3d_image` default plots the first element in the input data batch. And you can switch like below:

![image](https://user-images.githubusercontent.com/55491388/179914648-4e26fc10-7ff2-43ec-93de-185a953ab88a.png)
Since the "Tensorboard 3D" tab is not immediately available or an image and/or label is not yet displayed, wait a minute and then re-run it.
You can find more details from https://colab.research.google.com/github/KitwareMedical/tensorboard-plugin-3d/blob/main/demo/notebook/spleen_segmentation_3d.ipynb
Thanks!
